### PR TITLE
Updated annotators to 1.5.0, removed annotran which is now dead in the water.

### PR DESCRIPTION
**Pull request by ajrbyers**
- Updates to plugin object
- Removes annotran
- Could still use some tweaking but now workable

### CODE REVIEW

File: views.py [hunk 1]
```diff
@@ -1,27 +1,26 @@
...
 
-    enable_annotran = setting_handler.get_plugin_setting(plugin, 'enable_annotran', request.journal, create=True,
-                                                         pretty='Enable Annotran')
+def index(request):
+    plugin = plugin_settings.AnnotatorsPlugin.get_self()
 
-    enable_hypothesis = setting_handler.get_plugin_setting(plugin, 'enable_hypothesis', request.journal, create=True,
-                                                           pretty='Enable Hypothesis')
+    setting_group_name = 'plugin:annotators'
+    enable_hypothesis = setting_handler.get_setting(
```
Comment (mauromsl) [inline views.py]: This should use `setting_handler.get_plugin_setting` to avoid duplicating the logic here

**Review by mauromsl — CHANGES_REQUESTED** [review]
just one comment inline

**Review by mauromsl — APPROVED** [review]

**Review by joemull — APPROVED** [review]

**Comment by joemull** [comment]
@ajrbyers did this close #1 ?

**Comment by ajrbyers** [comment]
> @ajrbyers did this close #1 ?

It did. Nice. I want to use this plugin for my project-time work to allow a private annotator so althoug we could rename it "hypothesis" I think might be fine to leave it for now.

**Pull request merged**